crates/bevy_render/src/color/mod.rs

```diff
@@ -1377,29 +1380,17 @@ mod tests {
 
     #[test]
     fn hex_color() {
```

Comment:
Can you add one test that uses the `#` prefix just to make sure someone doesn't break this in the future.